region/crateria/central/Landing Site.json

```diff
@@ -825,25 +825,6 @@
         "Note that it is possible to jump all the way to the ledge above the bomb blocks, place a Power Bomb, then drop down, but it isn't typically easier."
       ]
     },
-    {
-      "id": 28,
-      "link": [3, 1],
-      "name": "Blue Space Jump (Top)",
-      "requires": [
-        {"notable": "Blue Space Jump (Top)"},
-        "canBlueSpaceJump",
-        "canTrickyDashJump",
-        {"getBlueSpeed": {"usedTiles": 30, "openEnd": 2}}
-      ],
-      "clearsObstacles": ["A"],
-      "note": [
-        "This strat is fairly precise. All jumps should be as small as possible.",
-        "Using at least one-tap short charge, jump off the ledge and descend as much as possible while keeping space jump active.",
-        "Once approaching the wall, be sure to jump again to continue through all of the bomb blocks.",
-        "If HiJump is available, it is much easier to use the blue space jump strat from the ground."
-      ],
-      "devNote": "Although there is a larger runway available, minimizing the speed is important."
```

Comment:
this strat was kinda useless and especially shouldn't be notable. I combined it with the other strat as a note for an alternate variant instead